specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2024-04-01/extensions.json

```diff
@@ -0,0 +1,761 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-05-01",
```

Comment:
The API version is not matched with the folder, should it be "2024-04-01"?

Reply:
fixed